app/ts/components/ConnectToWallet.tsx

```diff
@@ -24,15 +24,14 @@ export const ConnectToWallet = () => {
 				</Wrapper>
 			)
 		case 'disconnected': {
-			const connectToWallet = accountStore.value.connect
 			return (
 				<Wrapper>
 					<div class='grid grid-cols-[minmax(auto,max-content)_minmax(auto,max-content)_minmax(auto,max-content)] gap-3 items-center justify-center'>
 						<div class='leading-tight text-right'>Get started quickly by connecting your wallet</div>
 						<div class='transition animate-bounce-x'>
 							<Icon.ArrowRight />
 						</div>
-						<Button class='whitespace-nowrap' onClick={() => connectToWallet()}>
+						<Button class='whitespace-nowrap' onClick={accountStore.value.connect.bind(null, false)}>
```

Comment:
For the record, I liked `() => accountStore.value.connect()` better than the `bind` solution.  However, I'm fine with all of the options, so pick the one you like best.

I think `undefined` for first parameter of `bind` is more correct than `null` though?
```suggestion
						<Button class='whitespace-nowrap' onClick={accountStore.value.connect.bind(undefined, false)}>
```

Reply:
i'm a fan of arrow functions in event handlers as well but for some reason the type for `accoutStore.value` seem to have lost context as a return value `() => accountStore.value.connect()` in this switch statement.
<img width="683" alt="Screenshot 2023-02-13 at 2 43 11 AM" src="https://user-images.githubusercontent.com/1169838/218330442-72539de2-a050-4e79-b56b-4befb91bcaa2.png">

Reply:
Ah, yeah.  that makes sense.  This is why you needed the local, that makes sense.